Source/DBNetworkStackError.swift

```diff
@@ -65,24 +65,31 @@ extension DBNetworkStackError : CustomDebugStringConvertible {
     public var debugDescription: String {
         var result = ""
         
+        func appendData(result: inout String, data: Data?) {
```

Comment:
I would refactor the whole `CustomDebugStringConvertible `

We should talk about it later 🤓